packages/utils/src/lib/collect/implementation/md/list.ts

```diff
@@ -0,0 +1,9 @@
+export type Order = 'unordered' | 'checkbox';
+
+/**
+ * - {text}
+ */
+export function li(text: string, order: Order = 'unordered'): string {
+  const style = order === 'unordered' ? '-' : '- [ ]';
+  return `${style}  ${text}`;
```

Comment:
Produces unnecessary extra space:

```suggestion
  return `${style} ${text}`;
```